### PR TITLE
Fix Tizen Studio issue 469

### PR DESCRIPTION
**Pull request by mijin85cho**
Signed-off-by: Mijin, Cho <mijin85.cho@samsung.com>

### Change Description ###

Fix Tizen Studio issues reported on https://github.com/Samsung/tizen-docs/issues/469

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/508

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/508

File: docs/application/tizen-studio/native-tools/ui-builder/ui-builder.md [hunk 1]
```diff
@@ -124,21 +124,21 @@ void view1_button1_onclicked(uib_view1_view_context *vc, Evas_Object *obj, void
...
-      struct _uib_view1_control_context *cc;   
-
-      /* UI components in this view */    
-      Evas_Object *grid1;    
-      Evas_Object *button1;    
-      Evas_Object *check1;
-    };
-    typedef struct _uib_view1_view_context;
+    /* Parent evas_object which was a parameter of the create function */
+    Evas_Object *parent;
```
Comment (mijin80lee) [inline docs/application/tizen-studio/native-tools/ui-builder/ui-builder.md]: line128: Remove 3 spaces at the end.

File: docs/application/tizen-studio/release-notes/2-3-release-notes.md [hunk 2]
```diff
@@ -27,7 +27,7 @@
 	-   ST (Smart Things) certification is simplified by enabling auto certificate generation using Samsung Account sign-in.
```
Comment (mijin80lee) [inline docs/application/tizen-studio/release-notes/2-3-release-notes.md]: Missing the Release Date: Mar 30, 2018.

File: docs/application/tizen-studio/setup/install-sdk.md [hunk 3]
```diff
@@ -49,21 +49,23 @@ To install the Tizen Studio using the CLI installer:
...
 
    **Table: Install options**
+
    | Option             | Description                              |
    |--------------------|------------------------------------------|
    | `--show-license`   | Displays the Tizen Studio software license agreement.<br/> You must use this option alone. Do not use with other options. |
    | `--accept-license` | Accepts the license terms.               |
    | `--no-java-check`  | Skips the Java version check.            |
 
    **Table: Install command parameters**
```
Comment (mijin80lee) [inline docs/application/tizen-studio/setup/install-sdk.md]: line67: Is this table okay with the different browser such as firefox?

File: docs/application/tizen-studio/web-tools/config-editor.md [hunk 4]
```diff
@@ -1765,22 +1716,20 @@ The maximum length of the attribute and the element (except `<tizen:metadata>`,
...
-			<pre><code>
-&lt;tizen:ime&gt;
+			<pre><code>&lt;tizen:ime&gt;
    &lt;tizen:uuid&gt;6135122a-a428-40d2-8feb-a75f462c202c&lt;/tizen:uuid&gt;
    &lt;tizen:languages&gt;
       &lt;tizen:language&gt;en-us&lt;/tizen:language&gt;
       &lt;tizen:language&gt;de-de&lt;/tizen:language&gt;
    &lt;/tizen:languages&gt;
-&lt;/tizen:ime&gt;
+&lt;/tizen:ime&gt;
```
Comment (mijin80lee) [inline docs/application/tizen-studio/web-tools/config-editor.md]: line1730: Remove 2 spaces at the end.

File: docs/application/tizen-studio/web-tools/config-editor.md [hunk 5]
```diff
@@ -1545,12 +1508,10 @@ The maximum length of the attribute and the element (except `<tizen:metadata>`,
 				</li>
```
Comment (mijin80lee) [inline docs/application/tizen-studio/web-tools/config-editor.md]: line1506: Note form check.

File: docs/application/tizen-studio/setup/install-sdk.md (hunk 3 above)
Comment (mijin85cho) [reply docs/application/tizen-studio/setup/install-sdk.md]: No, it should show same layout on other browsers, too.
I fixed the tag. Would you please check the updated?

File: docs/application/tizen-studio/web-tools/config-editor.md (hunk 4 above)
Comment (mijin85cho) [reply docs/application/tizen-studio/web-tools/config-editor.md]: Done.

File: docs/application/tizen-studio/web-tools/config-editor.md (hunk 5 above)
Comment (mijin85cho) [reply docs/application/tizen-studio/web-tools/config-editor.md]: Done.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/508

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/508

**Pull request merged**